src/ServicePulse.Host/app/js/services/factory.rxjs.js

```diff
@@ -0,0 +1,8 @@
+; (function (window, angular, undefined) {
+    'use strict';
+
+    angular.module('sc')
+        .service('rx', function () {
```

Comment:
> But semantically I read it as factory.

how so? `Rx` is a constant provided by rxjs which we define here for property injection reasons. I don't see what type of code you would add here at any time.